### DISCUSSION TITLE
Reloading the route which included in prerender.ignore not being loaded properly on production (target: 'static',)

### DISCUSSION THREAD

**Discussion by muhammad-grayphite** [nuxt/nuxt#21444] [Questions]
Actually there are some admin pages that I dont want to prerender, here my configuration in nuxt.config file
```
target: 'static',   
 nitro: {
      prerender: { 
        crawlLinks: true, 
        ignore: [
          '/admin', // path starts with /admin
          '/customers', // path starts with /customers
          '/attendance',
          '/reservation',
          '/profile',
          '/belts',
          '/customer-belts'
        ],
      }
    }
```
this is working fine these routes are being igored on build time. Now when I go to one of these pages lets say '/attendance' using $router.push its working as expected but on production when I refresh the same page or go directly `https://{domain}/attendance` it shows blank page. I am using the pinia store and its not even loading the store object and giving error.

I wanted to discuss is there anything I can do in this case? or I need to remove these routes from ignore to make it work?

Many Thanks!

**Comment by muhammad-grayphite** [comment]
Any update on this?

**Comment by m-haziq** [comment]
I'm getting this issue on dynamic slug pages.
